File: datasets/IR-Plag/convertai-original-java-rest-python/case-06/plagiarized/L6/06/TestSatuArray.py
import sys

def input_arr(arr):
    arr2 = [0] * 10
    for x in range(10):
        arr[x] = int(input("Read a number: "))
    for i in range(10):
        arr2[i] = arr[9 - i]
    return arr2

def print_arr(arr):
    for x in range(10):
        print(arr[x])

def main():
    arr = [0] * 10
    print_arr(input_arr(arr))

if __name__ == "__main__":
    main()

File: datasets/IR-Plag/intellij-original-java-rest-kotlin/case-06/plagiarized/L6/06/TestSatuArray.kt
package IR

import java.util.*

object TestSatuArray {
    @JvmStatic
    fun main(args: Array<String>) {
        val sc = Scanner(System.`in`)
        val arr = IntArray(10) //inisialisasi array

        printArr(inputArr(arr))
    }

    fun inputArr(arr: IntArray): IntArray {
        val sc = Scanner(System.`in`)
        var x = 0
        val arr2 = IntArray(10)
        while (x < 10) { //looping untuk input array
            print("Read a number: ")
            arr[x] = sc.nextInt() //input ke array
            x++
        }
        x = 9
        for (i in 0..9) {
            arr2[i] = arr[x]
            x--
        }
        return arr2
    }

    fun printArr(arr: IntArray) {
        var x = 0
        while (x < 10) { //looping untuk menampilkan array
            println(arr[x]) //menampilkan array
            x++
        }
    }
}
